src/config/users-and-groups.md

```diff
@@ -0,0 +1,39 @@
+# Users and groups
+
+## Default groups
+
+Void Linux defines a number of groups by default.
+
+| Group      | Description                                                   |
+|------------|---------------------------------------------------------------|
+| `root`     | Complete access to the system.                                |
+| `bin`      | Unused - present for historical reasons.                      |
+| `sys`      | Unused - present for historical reasons.                      |
+| `kmem`     | Ability to read from `/dev/mem` and `/dev/port`.              |
+| `wheel`    | Elevated privileges for specific system administration tasks. |
+| `tty`      | Access to TTY-like devices:                                   |
+|            | `/dev/tty*`, `/dev/pts*`, `/dev/vcs*`.                        |
+| `tape`     | Access to tape devices.                                       |
+| `daemon`   | System daemons that need to write to files on disk.           |
+| `floppy`   | Access to floppy drives.                                      |
+| `disk`     | Raw access to `/dev/sd*` and `/dev/loop*`.                    |
+| `lp`       | Access to printers.                                           |
+| `dialout`  | Access to serial ports.                                       |
+| `audio`    | Access to audio devices.                                      |
+| `video`    | Access to video devices.                                      |
+| `utmp`     | Ability to write to `/var/run/utmp`, `/var/log/wtmp`          |
+|            | and `/var/log/btmp`.                                          |
+| `adm`      | Unused - present for historical reasons. This group was       |
+|            | traditionally used for system monitoring, such as viewing     |
+|            | files in `/var/log`.                                          |
+| `cdrom`    | Access to CD devices.                                         |
+| `optical`  | Access to DVD/CD-RW devices.                                  |
+| `mail`     | [Mailboxes in /var/mail?]                                     |
```

Comment:
is this a question?

Reply:
Sorry, yes. The information i found suggested this was what the `mail` group is for, but i wasn't sure whether this is true on Void, or whether it's actually unused and is only present for historical reasons.

Reply:
I'm really not sure on this one. It's definitely historical, but could be still used by some mail packages? I have no idea.

Reply:
@bobertlo: Fair enough. :-)
@Duncaen: Are you aware of the `mail` group being used by any Void package(s)?

Reply:
Looks like dma is the only package that uses the mail account, all others setup their own dedicated users.

Reply:
@Duncaen: Thanks, PR updated accordingly.